data_pipeline/scraper/scraper.py

```diff
@@ -117,8 +118,27 @@ def downloadExcel(self, driver):
                 else:
                     downloadLinkElement.click()
                     count += 1
+                    
+                    while(1):
+                        if os.path.exists('./data/transactionExportGrid.xls'):
+                            countFile += 1
+                            renamedFile = './data/transactionExportGrid' + '(' + str(countFile) + ').xls'
+                            os.rename('./data/transactionExportGrid.xls', renamedFile)
+                            break
+                        sleep(0.1)
+                    
+                    """
```

Comment:
You can delete the commented out code, then looks good to submit.

Reply:
Got it! Sorry, I somehow didn't realize that I left it commented there